### ISSUE TITLE
Support for pixelfed?

### ISSUE THREAD

**Issue by kr428** [TwidereProject/Twidere-Android#1341]
Do you think it is possible to add support for pixelfed (https://pixelfed.org/) to Twidere? So far, it seems both tusky and fedilab provide that as pixelfed seems to (roughly) implement the Mastodon mobile client API. Would be nice to have pixelfed in here too. :) Thanks for eventually considering, as well as for the great work in Twidere so far!

**Comment by iknownothingaboutcomputers** [comment]
Android and iOS apps for PixelFed are being developed currently. 

Source: PixelFed's Twitter a/c.

**Comment by kr428** [comment]
Yes I know. 🙂 However having this in Twidere would help reducing the amount of different apps on the device that all do roughly the same. 😉

**Comment by iknownothingaboutcomputers** [comment]
@kr428 But it'd increase the workload of the team maintaining Twidere, wouldn't it? I don't think they're a big team. Got enough on their plate as it is.
